tests/tracer/test_trace_utils.py

```diff
@@ -522,73 +522,53 @@ def test_set_http_meta_case_sensitive_headers_notfound(mock_store_headers, span,
     mock_store_headers.assert_called()
 
 
-@mock.patch("ddtrace.contrib.trace_utils._store_headers")
 @pytest.mark.parametrize(
-    "header_env_var,headers_dict,expected_keys,expected",
+    "header_env_var,headers_dict,expected",
     [
         (
             "",
             {"x-forwarded-for": "8.8.8.8"},
-            ["runtime-id", "network.client.ip", http.CLIENT_IP],
             "8.8.8.8",
         ),
         (
             "",
             {"x-forwarded-for": "8.8.8.8,127.0.0.1"},
-            ["runtime-id", "network.client.ip", http.CLIENT_IP],
             "8.8.8.8",
         ),
         (
             "",
             {"x-forwarded-for": "192.168.144.2"},
-            ["runtime-id", "network.client.ip", http.CLIENT_IP],
             "192.168.144.2",
         ),
         (
             "",
             {"x-forwarded-for": "127.0.0.1"},
-            ["runtime-id", "network.client.ip", http.CLIENT_IP],
             "127.0.0.1",
         ),
         (
             "",
             {"x-forwarded-for": "192.168.1.14,8.8.8.8,127.0.0.1"},
-            ["runtime-id", "network.client.ip", http.CLIENT_IP],
             "8.8.8.8",
         ),
         (
             "",
             {"x-forwarded-for": "192.168.1.14,127.0.0.1"},
-            ["runtime-id", "network.client.ip", http.CLIENT_IP],
             "192.168.1.14",
         ),
-        ("", {"x-forwarded-for": "foobar"}, ["runtime-id"], None),
-        ("via", {"x-forwarded-for": "4.4.8.8"}, ["runtime-id"], None),
-        ("via", {"via": "8.8.8.8"}, ["runtime-id", "network.client.ip", http.CLIENT_IP], "8.8.8.8"),
+        ("", {"x-forwarded-for": "foobar"}, ""),
+        ("via", {"x-forwarded-for": "4.4.8.8"}, ""),
+        ("via", {"via": "8.8.8.8"}, "8.8.8.8"),
         (
             "via",
             {"x-forwarded-for": "4.4.4.4", "via": "8.8.4.4"},
-            ["runtime-id", "network.client.ip", http.CLIENT_IP],
             "8.8.4.4",
         ),
     ],
 )
-def test_set_http_meta_headers_ip(
```

Comment:
do we have tests elsewhere that the appropriate tags get adds to spans when we have a request ip ?

Reply:
Some of the `test_set_http_meta_*`  in this file and in `test_django_appsec` and some other tests in django and flask. In this case the test wants to check the `_get_request_header_client_ip` which doesn't set any tags thus I removed them and the call of `set_http_meta` that didn't make sense for testing it (it was also needed for some unrelated thing related to the changes in these PRs that doesn't come to my mind now).